src/lib.rs

```diff
@@ -64,8 +74,26 @@ pub fn drop_table(table: &str) -> io::Result<()> {
     Ok(())
 }
 
+#[allow(unused_must_use)]
```

Comment:
I'll have to double check when I get off work; I could have sworn I took it
off and didn't get a warning.
On Jun 14, 2016 09:02, "Regis Boudinot" <notifications@github.com> wrote:

> In src/lib.rs
> <https://github.com/pindell-matt/fe_bucket/pull/7#discussion_r67001196>:
>
> > @@ -64,8 +74,26 @@ pub fn drop_table(table: &str) -> io::Result<()> {
> >      Ok(())
> >  }
> >
> > +#[allow(unused_must_use)]
>
> This warning absolutely fires off when using Racer
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pindell-matt/fe_bucket/pull/7/files/9fd65f447f762eaf32210c741f5d0919b1789a11..e40706def2b4330be6d641d24b18819d8fb28f26#r67001196>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEAKew4aU86unJXMrPCGBEpo1BF2uRdNks5qLtCEgaJpZM4I0zw8>
> .
>